quiche/src/lib.rs

```diff
@@ -1878,6 +1893,22 @@ impl Connection {
             stopped_stream_remote_count: 0,
         };
 
+        // Enable path MTU Discovery and start probing with the largest datagram
+        // size
+        if config.pmtud {
+            let path_pmtud = &mut conn.paths.get_mut(active_path_id)?.pmtud;
+            path_pmtud.should_probe(config.pmtud);
+            path_pmtud.set_probe_size(config.max_send_udp_payload_size);
+            path_pmtud.enable(config.pmtud);
+            trace!(
+                "PMTU configured pmtu {} next_size {} pmtu_probe {} enable {}",
+                path_pmtud.get_current(),
+                path_pmtud.get_probe_status(),
+                path_pmtud.get_probe_size(),
+                path_pmtud.is_enabled(),
+            );
```

Comment:
How useful is this trace line? It seems like it just echoes something that the application itself would know

Reply:
Implementing `std::fmt::Debug` would be helpful for logging purpose.